examples/grid.py

```diff
@@ -0,0 +1,59 @@
+import pygame as pg
+
+TITLE = "Grid"
+TILES_HORIZONTAL = 10
+TILES_VERTICAL = 10
+TILE_SIZE = 100
+WINDOW_WIDTH = 1000
+WINDOW_HEIGHT = 1000

```

Comment:
This window height will be too big for some people (like if they have display scaling from the OS). Maybe something like 800x800 would be safer.

Reply:
Good idea.